cmd/blazer.go

```diff
@@ -20,33 +22,35 @@ func Start() {
 		fmt.Println("Error parsing flags: ", err)
```

Comment:
We should try to make all the free-flowing err/info msgs as constant strings.

Reply:
sure, will update it

Reply:
please create an issue for it. we need to change in many places